Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -4434,67 +4433,67 @@
     ]
   },
   {
-    "Input": "Ya terminé todos los trabajos hace más de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "durante más de 2 semanas antes de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "hace más de 2 semanas",
+        "Text": "durante más de 2 semanas antes de hoy",
         "Type": "daterange",
-        "Start": 30,
-        "Length": 21
+        "Start": 0,
+        "Length": 37
       }
     ]
   },
   {
-    "Input": "Volveré dentro de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de 2 semanas a partir de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "dentro de 2 semanas",
+        "Text": "dentro de 2 semanas a partir de hoy",
         "Type": "daterange",
-        "Start": 8,
-        "Length": 19
+        "Start": 0,
+        "Length": 35
       }
     ]
   },
   {
-    "Input": "Volveré dentro de menos de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de menos de 2 semanas a partir de hoy",
```

Comment:
Don't remove other pieces of the sentence. If the input is "Volveré dentro de menos de 2 semanas", the response should be correct.

Reply:
Ideally all spec cases should be full sentences.

Reply:
> Ideally all spec cases should be full sentences.

will modfiy spec cases

Reply:
> Don't remove other pieces of the sentence. If the input is "Volveré dentro de menos de 2 semanas", the response should be correct.

will support it

Reply:
> Don't remove other pieces of the sentence. If the input is "Volveré dentro de menos de 2 semanas", the response should be correct.
"dentro de menos de 2 semanas" is not "dateperiod " is belong to "duration", my opinion is to remove this sentence and save the new sentence, thanks

Reply:
In that case it should be the full sentence "Volveré dentro de menos de 2 semanas a partir de hoy", right? Then we should make sure the duration spec includes the other case.